hackcon-2015/binary/LetsLearnDebugging/README.md

```diff
@@ -0,0 +1,17 @@
+# Hackcon 2015: Let's Learn Debugging
+
+**Category:** Binary
+**Points:** 75
+**Author:** [Sambhav Satija](https://github.com/darkryder)
+**Difficulty:** Medium
+**Description:** 
+
+The program apparently once could print more than stories.
+
```

Comment:
also try to add the link of the file in the description 

well it can be understood but the we do it in standard way, it's not a big deal